pkg/cli/options.go

```diff
@@ -0,0 +1,31 @@
+package cli
+
+import flag "github.com/spf13/pflag"
+
+type PaastaOptions struct {
+	Help bool
+	Verbosity int
+	SoaDir string
+	SysDir string
+	UseMesosCache bool
+}
+
+func (o *PaastaOptions) Setup() {
+	flag.BoolVarP(&o.Help, "help", "h", false, "")
+	flag.CountVarP(&o.Verbosity, "verbose", "v", "")
+	flag.StringVarP(&o.SoaDir, "soa-dir", "d", "/nail/etc/services", "")
+	flag.StringVarP(&o.SysDir, "sys-dir", "C", "/etc/paasta", "")
+	flag.BoolVarP(&o.UseMesosCache, "use-mesos-cache", "", false, "")
```

Comment:
do we really want to push this back on users?

Reply:
i wanted to keep the original interface